### DISCUSSION TITLE
Windows 11 version

### DISCUSSION THREAD

**Discussion by elerock** [SafeExamBrowser/seb-win-refactoring#315] [Q&A]
I'm going to take a test using SEB but my laptop is provided with windows 11.
Does the italian version of Safe Exam Browser work using a version superior to windows 10?

**Comment by dbuechel** [comment]
Yes, it should work as expected. Though we have to state that we were not yet able to perform a complete release testing with Windows 11, but we hope to be able to do so for the upcoming release version 3.3.2.
